plugin/info.xml

```diff
@@ -4,7 +4,7 @@
 	<type>plugin</type>
 
 	<name>AirConsole</name>
-	<version>1.3</version>
+	<version>1.5.0.3</version>

```

Comment:
Just tested the c2addon in construct2.
Can you set it to 1.4.0. Then I can merge and release it